src/components/data-management/ProjectDetails.jsx

```diff
@@ -104,6 +115,47 @@ const ProjectDetails = ({ width, height }) => {
     }
   }, [samples, activeProject]);
 
+  const downloadData = async (type) => {
+    const experimentId = activeProject?.experiments[0];
```

Comment:
I think the logic of choosing the experiment to download should not be inside this function it just does too many things inducing errors. This function should receive a type and the experimentID (or whatever it's needed) and then just download. The logic about which is the active experiment shouldn't be here. I suspect that this kind of accesses are the ones causing projects to interfere with one another. You click one experiment but the activeProject is not the correct one. If every project passes explicitly it's own ID this will not happen.

Reply:
Refactored to another function. It now takes experimentId as a parameter